.storybook/test-runner.ts

```diff
@@ -16,9 +27,9 @@ const config: TestRunnerConfig = {
 
     expect(content).toBeDefined();
 
-    // We strip the `class` tags from the HTML content as we do not want
-    // to pollute the snapshots with generated class names
-    expect(content.replace(/class="(.*?)"/gm, '')).toMatchSnapshot();
+    // We stripe specific HTML tags whose values are auto-generated or can change from deployment to deployment
```

Comment:
`stripe` is not correct.

`strip` is correct, but I'm suggesting a more accurate word

```suggestion
    // We remove specific HTML tags whose values are auto-generated or can change from deployment to deployment
```